helm_deploy/court-list-splitter/Chart.yaml

```diff
@@ -3,3 +3,11 @@ appVersion: "1.0"
 description: A Helm chart for Kubernetes
 name: court-list-splitter
 version: 0.1.0
+
+dependencies:
+  - name: generic-service
+    version: 1.0.9
+    repository: https://ministryofjustice.github.io/hmpps-helm-charts
+  - name: generic-prometheus-alerts
+    version: 0.1.4
+    repository: https://ministryofjustice.github.io/hmpps-helm-charts   
```

Comment:
I noticed that CHER project also had this prometheus dependency so added this also as must come from the bootstrapping